cli/tools/upgrade.rs

```diff
@@ -47,6 +47,19 @@ trait UpdateCheckerEnvironment: Clone + Send + Sync {
   fn current_time(&self) -> chrono::DateTime<chrono::Utc>;
 }
 
+// Identical to chrono::Utc::now() but without the system "clock"
+// feature flag.
+pub fn utc_now() -> chrono::DateTime<chrono::Utc> {
```

Comment:
1. Share this code between bench and cli: `cli/utc_now.rs`

2. Expand the comment about the purpose of this implementation (i.e. that we're minimizing the dylibs that we link to)

Reply:
Maybe `cli/util/time.rs`